File: 2.Het/Szülő-gyerek/szulo-gyerek.cpp
#include <iostream>

using namespace std;

class Szulok{
public: void gyerekek(){cout << "szoveg\n";};
};

class Gyerek : public Szulok{
	public: void testverek(){cout << "pelda\n";};
};

int main ( int argc, char **argv )
{
Szulok szulo;
szulo.gyerekek();
//szulo.testverek();
return 0;
}

File: 2.Het/Szülő-gyerek/szulo_gyerek.java
class Szulok{
	void gyerekek(){System.out.println("szoveg");}
	void stop(){System.out.println("szoveg");}
}

class Gyerekek extends Szulok{
	void testverek(){System.out.println("pelda");}
	void stop(){System.out.println("pelda");}
}

class szulo_gyerek{

	public static void main(String[] args){
		Szulok szulo = new Gyerekek();
		szulo.gyerekek();
		szulo.stop();
		//szulo.testverek();
		
	}
}
